### DISCUSSION TITLE
Quota exceeded? (gpt 3.5)

### DISCUSSION THREAD

**Discussion by yurygl** [nhaouari/obsidian-textgenerator-plugin#165] [Q&A]
Hello all, I just installed the plugin but when I try to use it, it just says I have exceeded my quota, and that I need to check my plan and billing status. I do have an active GPT plus account, but also I am just trying to use 3.5 :/ Could someone please suggest what the issue might be?

**Comment by QSBRI-RossB** [comment]
Hi, I have the same issue and now it looks like Chat GPT API is now a paid service. 
https://openai.com/pricing#language-models
